docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -71,13 +71,14 @@ They track billable operations within the Temporal Cloud Service, such as starti
 - **Search Attribute upsert requested**.
   Occurs after a Workflow starts and invokes `UpsertSearchAttributes`.
 - **Signal sent**.
-  Includes sending a [Signal](/encyclopedia/workflow-message-passing#sending-signals) from a client or from within a Workflow to another Workflow.
-- **Query received**. [Queries](/encyclopedia/workflow-message-passing#sending-queries) aren't recorded in Event History.
-  An operation such as viewing the call stack in the Temporal Cloud UI results in a Query.
+  An Action occurs for every [Signal](/encyclopedia/workflow-message-passing#sending-signals), whether sent from a Client or from a Workflow.
+- **Query received by Worker**.
```

Comment:
@drewhoskins-temporal Is this correct? As I understand it, workflows dequeue tasks and execute them. So is the action recorded when the query hits the Temporal Service, or only when the Worker picks it up?